src/devices/Ws28xx/Ws28xx.sln

```diff
@@ -1,13 +1,13 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26124.0
+# Visual Studio Version 16
```

Comment:
NIT: Can we please undo the changes into this file?

Reply:
I'll undo those changes.